### PR TITLE
cleanup smsbillables logging

### PR DESCRIPTION
**Pull request by nickpell**
seems like a good idea to actually log logs

@benrudolph

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1036,8 +1036,8 @@
             'propagate': False,
         },
         'smsbillables': {
-            'handlers': ['file'],
-            'level': 'ERROR',
+            'handlers': ['file', 'console', 'mail_admins'],
```
Comment (benrudolph) [inline settings.py]: does this mean every INFO level log message will be sent to us as email?

**Comment by benrudolph** [comment]
haha meta :8ball:

File: settings.py (hunk 1 above)
Comment (nickpell) [inline settings.py]: poop, you're right.  will separate out.

Comment (nickpell) [inline settings.py]: actually I think I had this right the first time.  ```mail_admins``` is set to level ```ERROR```

Comment (nickpell) [inline settings.py]: confirmed on staging that this is working (see email error logs)

```
>>> from corehq.apps.smsbillables.utils import log_smsbillables_error, log_smsbillables_info
>>> log_smsbillables_info('this is a test. shouldnt get email')
2016-03-21 16:04:52,322 INFO [SMS Billables] this is a test. shouldnt get email
>>> log_smsbillables_error('this is a test. should get email')
2016-03-21 16:05:04,857 ERROR [SMS Billables] this is a test. should get email
```

Comment (benrudolph) [inline settings.py]: cool thanks for checking

**Pull request merged**